udata/tasks.py

```diff
@@ -26,6 +29,7 @@ def __call__(self, *args, **kwargs):
 
 class JobTask(ContextTask):
     abstract = True
+    default_queue = 'low'
```

Comment:
Maybe `default`? Because, you know, `default` ;-)

Reply:
This is `default` for tasks, jobs are meant to be long running tasks this is why I set them to `low` by default. But I can undo this, but this mean we have to declare `queue=low` manually for every job

Reply:
Makes senses 👍